OpenSim/Examples/ExampleMain/OutputReference/TugOfWar_Complete.cpp

```diff
@@ -71,7 +71,10 @@ int main()
 
         // GROUND BODY
         // Get a reference to the model's ground body
-        OpenSim::Body& ground = osimModel.getGroundBody();
+        OpenSim::Body& ground = *new OpenSim::Body("ground", SimTK::Infinity,
```

Comment:
Is the need for this Body going to be removed by #369? If so, can there be a note made for this so we don't forget? Otherwise, I don't think our examples should have `*new`.